src/index.html

```diff
@@ -6,9 +6,24 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles/index.css" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat" />
-    <title>Web App</title>
+    <title>Trainee Program</title>
   </head>
   <body>
-    <h1>Hello, World!</h1>
+    <div class = "header">
```

Comment:
PR ta ok. Só remover esses espaçamentos entre alguns atributos
```suggestion
    <div class="header">
```